concert/base.py

```diff
@@ -597,7 +628,10 @@ def stash(self):
         if not self.writable:
             raise ParameterError("Parameter `{}' is not writable".format(self.name))
 
-        self._saved.append(self.get().result())
+        if self.target_readable:
+            self._saved.append(self.get_target().result())
+        else:
+            self._saved.append(self.get().result())
```

Comment:
Doesn't this discard what user sets? What if target sais -5 mm for the lower limit and I set it to -3 mm, then stash will not save what I specified, right?

Reply:
Not really. The get_target() should give the last value that was set. In the implementation this can stored in the Quantity or Device, or it can be read from the physical device (e.g. tango you can ask what the last set value for a parameter was).